### PR TITLE
doma: update vdispl and vkb configs for CES2018 demo

### PR DESCRIPTION
**Pull request by al1img**
Signed-off-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>

### CODE REVIEW

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-h3.cfg [hunk 1]
```diff
@@ -98,8 +98,8 @@ vcpus = 4
 # docs/misc/xl-network-configuration.markdown
 vif = [ 'backend=DomD,bridge=xenbr0,mac=08:00:27:ff:cb:ce' ]
 
-vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1024x768' ]
+vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1920x1080;1001:1088x1487' ]
```
Comment (aanisov) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-h3.cfg]: Are you sure about an odd height of a secondary screen?

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-m3.cfg [hunk 2]
```diff
@@ -78,8 +78,8 @@ vcpus = 4
 # docs/misc/xl-network-configuration.markdown
 vif = [ 'backend=DomD,bridge=xenbr0,mac=08:00:27:ff:cb:ce' ]
 
-vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1024x768' ]
+vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1920x1080;1001:1088x1487' ]
```
Comment (aanisov) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-m3.cfg]: Are you sure about an odd height of a secondary screen?

Comment (arminn) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-m3.cfg]: I'm also curious about 1487 height.

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-h3.cfg (hunk 1 above)
Comment (al1img) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-h3.cfg]: Yes, this is size of application AGL layout.

**Comment by al1img** [comment]
Comments added

**Review by aanisov — APPROVED** [review]

**Review by andr2000 — APPROVED** [review]

**Pull request merged**